### ISSUE TITLE
Not working with Fluent

### ISSUE THREAD

**Issue by RVXD** [littlegiant/silverstripe-seo-editor#13]
The handleAction function does a direct UPDATE on the database to gain speed:

```
DB::query("UPDATE SiteTree SET {$fieldName} = '{$sqlValue}' WHERE ID = {$page->ID}");
if ($page->isPublished()) {
     DB::query("UPDATE SiteTree_Live SET {$fieldName} = '{$sqlValue}' WHERE ID = {$page->ID}");
}
```
This causes other modules like Fluent to work incorrectly. 
If you change this to the ORM method:

```
$page->$fieldName = $sqlValue;
$page->write();
if ($page->isPublished()) {
      $page->doPublish();
}
```
I made a fork with a new branch called enable-augmentation, with this implemented:
https://github.com/RVXD/silverstripe-seo-editor/tree/enable-augmentation

Perhaps if you would want the increased speed for sites without Fluent, this could be made optional.

**Comment by stevie-mayhew** [comment]
Hi @RVXD ,

We made this choice to directly alter the database rather than force a publish cycle as there may be other changes which are waiting to be published and we didn't want to assume that we could publish the page.

We've actually got a Fluent compatible version of the module. I've just pushed this up to github for you here: https://github.com/littlegiant/silverstripe-seo-editor-fluent

We will likely look at making it extensible rather than having to have two separate repositories as part of the upgrade work for SilverStripe 4.
